src/util.js

```diff
@@ -131,11 +131,11 @@ export function className(elem, mod) {
  * @returns {Array}
  */
 export function toArray(list) {
-	return Array.prototype.slice.call(list);
+	return Array.from(list);
```

Comment:
Array.from isn’t supported by IE and Opera, otherwise I wouldn’t write my own function.